### PR TITLE
Ssdk 18.0.1 update

### PR DESCRIPTION
**Pull request by MalarsriA**
Upgrade to 18.0.1 (including changes for previous ssdk breaking changes)
CODEOWNERS - included befound team , as we are the maintainers of this repo and to make the review process faster on minor changes and upgrades

### CODE REVIEW

**Review by jjjasper — APPROVED** [review]

File: services/pom.xml [hunk 1]
```diff
@@ -20,8 +20,8 @@
 
     <properties>
         <properties-maven-plugin.version>1.0.0</properties-maven-plugin.version>
-        <testcontainers.version>1.19.1</testcontainers.version>
-        <boat-maven-plugin.version>0.17.24</boat-maven-plugin.version>
+        <testcontainers.version>1.19.7</testcontainers.version>
```
Comment (uesleilima) [inline services/pom.xml:23]: spring boot now manages the testcontainers dependencies version, I would remove any explicit version to it or any of its dependencies

**Review by uesleilima — CHANGES_REQUESTED** [review]

File: TESTCONTAINERS.md [hunk 2]
```diff
@@ -155,19 +155,19 @@ For multiple databases testing we can use different profiles:
...
 If the init script path is prefixed file:, it will be loaded from a file (relative to the working directory, which will usually be the project root).
 
-`jdbc:tc:mysql:5.7.22:///databasename?TC_INITSCRIPT=file:src/main/resources/init_mysql.sql`
+`jdbc:tc:mysql:8.0.22:///databasename?TC_INITSCRIPT=file:src/main/resources/init_mysql.sql`
 
 ##### Using an init function
 Instead of running a fixed script for DB setup, it may be useful to call a Java function that you define. This is intended to allow you to trigger database schema migration tools. To do this, add TC_INITFUNCTION to the URL as follows, passing a full path to the class name and method:
 
-`jdbc:tc:mysql:5.7.22:///databasename?TC_INITFUNCTION=org.testcontainers.jdbc.JDBCDriverTest::sampleInitFunction`
+`jdbc:tc:mysql:8.0:///databasename?TC_INITFUNCTION=org.testcontainers.jdbc.JDBCDriverTest::sampleInitFunction`
```
Comment (ugur-bb) [inline TESTCONTAINERS.md:170]: We use `8.0.36` as per this RND ticket: https://backbase.atlassian.net/browse/RND-420

Comment (MalarsriA) [reply TESTCONTAINERS.md]: updated the PR

File: services/pom.xml (hunk 1 above)
Comment (MalarsriA) [reply services/pom.xml]: Removed it @uesleilima

File: services/product/pom.xml [hunk 3]
```diff
@@ -113,8 +113,8 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>com.backbase.buildingblocks</groupId>
-            <artifactId>bb-activemq-broker</artifactId>
+            <groupId> org.apache.activemq</groupId>
```
Comment (ugur-bb) [inline services/product/pom.xml:116]:
```suggestion
            <groupId>org.apache.activemq</groupId>
```

File: services/review/pom.xml [hunk 4]
```diff
@@ -91,8 +91,8 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>com.backbase.buildingblocks</groupId>
-            <artifactId>bb-activemq-broker</artifactId>
+            <groupId> org.apache.activemq</groupId>
```
Comment (ugur-bb) [inline services/review/pom.xml:94]:
```suggestion
            <groupId>org.apache.activemq</groupId>
```

File: services/store/pom.xml [hunk 5]
```diff
@@ -97,8 +97,8 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>com.backbase.buildingblocks</groupId>
-            <artifactId>bb-activemq-broker</artifactId>
+            <groupId> org.apache.activemq</groupId>
```
Comment (ugur-bb) [inline services/store/pom.xml:100]:
```suggestion
            <groupId>org.apache.activemq</groupId>
```

File: services/pom.xml [hunk 6]
```diff
@@ -20,8 +20,7 @@
 
     <properties>
         <properties-maven-plugin.version>1.0.0</properties-maven-plugin.version>
-        <testcontainers.version>1.19.1</testcontainers.version>
```
Comment (uesleilima) [inline services/pom.xml:23]: if you removed this here you need also to remove the `testcontainers-bom` from the dependency manager section

**Review by uesleilima — CHANGES_REQUESTED** [review]

**Review by uesleilima — APPROVED** [review]

**Review by mahdieha — APPROVED** [review]

**Pull request merged**